src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+
+<Callout variant="tip">
+You have two choices for exporting data from your application to New Relic via OTLP:
+* Directly from your app
+* Via an OpenTelemetry collector
+
+This guide covers the first option. If you wish to export your data via a collector, check out this [documenation](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-basic/) for details.
+</Callout>
+## Requirements [#requirements]
+
+Before you get started, make sure you have the following:
+
+* A New Relic account: Sign up for a [free account](https://newrelic.com/signup) if you don't already have one.
+* Java 17 or higher: If you don't already have this, see [these download options](https://www.java.com/en/download/manual.jsp).
+
+## Run our pre-instrumented program [#pre-instrum]
+
+This is a great option if you want us do the instrumentation so you can quickly see what's it's like to send data to New Relic and view it in our UI.
+
+1. Clone the demo app and navigate to the following directory:
+    ```
+    git clone git@github.com:newrelic/newrelic-opentelemetry-examples.git
+    cd newrelic-opentelemetry-examples/getting-started-guides/java/Instrumented
+    ```
+2. In your terminal, set these two environment variables to send data to your New Relic account:
+    ```
+    export OTEL_EXPORTER_OTLP_HEADERS=api-key=INSERT_YOUR_NEW_RELIC_LICENSE_KEY
+    export OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317
+
+    ```
+    * Make sure to use your [ingest license key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key).
+    * If your account is based in the EU, set the endpoint to: https://otlp.eu01.nr-data.net:4317
+
+3. Set this environment variable to name the demo app:
+    ```
+    export OTEL_SERVICE_NAME=getting-started-java
+    ```
+4. In the `java/Instrumented/` directory, run the application:
+    ```
+    ./gradlew bootRun
+    ```
+5. Open a new terminal and exercise the application from the `Instrumented/java` directory by running this command:
+    ```
+    ./load-generator.sh
+    ```
+
+    <Callout variant="tip">
+      Alternatively, you can reach the endpoint in the browser at this URL: `http://localhost:8080/fibonacci?n=INSERT_A_VALUE`. Replace `INSERT_A_VALUE` with value from 1 to 90. To generate an error, insert an integer outside the valid range.
+    </Callout>
+
+6. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & services**.
+7. Click your new entity (service) called `getting-started-java` and explore the UI. For more tips about what to look for in the UI, see [View your data in New Relic](#view-data).
+8. When you're finished looking at your data in the UI, shut down the application by pressing CONTROL+C in both terminal sessions.
+
+## Use the OpenTelemetry Java agent [#java-agent]
```

Comment:
I wonder if maybe we might put 'Tutorial #2:' or something preceding these major headers to better communicate: these are kind of separate tutorials/processes.

Reply:
@zuluecho9 Great idea! I added "Tutorial X:" in front of each main H2 and also added this to the mini-TOC at the top.